src/constants.py

```diff
@@ -0,0 +1,15 @@
+DRY_BASES = "dry_base"
+WET_BASES = "wet_base"
+SWEETENERS = "sweetener"
+FLAVORINGS = "flavoring"
+SPICES = "spices"
+FILLINGS = "filling"
+LEAVENERS = "leavener"
+TOPPINGS = "toppings"
+
+CUP = "cup"
+OUNCE = "oz"
+TSP = "tsp"
+TBSP = "tbsp"
+
+CATEGORIES = [DRY_BASES, WET_BASES, SWEETENERS, FLAVORINGS, SPICES, FILLINGS, LEAVENERS, TOPPINGS]
```

Comment:
while i think this is fine, I think python also has enum classes that we can define but this does the same thing so for what we need it is probably fine.